chat/main.py

```diff
@@ -0,0 +1,42 @@
+import streamlit as st
+
+from embedding_db import EmbeddingDatabase
+from embedding_model import EmbeddingModel
+from llm import LargeLanguageModel
+from rag import QuestionAnsweringRAG
+from dotenv import load_dotenv, find_dotenv
+
+
+# Initialize models and RAG
+load_dotenv(find_dotenv())
```

Comment:
Do we need dotenv? I find it usually adds complexity for for good reason

Reply:
Context https://mlops-community.slack.com/archives/C06UAQ3AVM1/p1729255049359939?thread_ts=1729250251.042499&cid=C06UAQ3AVM1